### DISCUSSION TITLE
Swig development - Golang

### DISCUSSION THREAD

**Discussion by tamama** [swig/swig#2053] [General]
There are a couple of PRs regarding Swig-Go, including mine. These have been around for some time.

May I ask who is product owner for Swig-Go?

I could do a quick demo on how mine works.

**Comment by ojwb** [comment]
The PR in question seems to be #2041.

SWIG is a volunteer-run project, so it can take time for someone to get to reviewing PRs or responding to issues (this is true for most FOSS projects in my experience; TBH my experience with proprietary software isn't any better - even if you have a support contract with a time-to-respond written in, it typically takes much longer to get a **useful** response).

Your PR did get a review, but it seems not until after you'd got impatient and closed it.  If you're still wanting to get that PR merged, please address the review comments and reopen it.

**Comment by tamama** [comment]
Hi @ojwb , thanks for the update. 

After 2 years, Golang is currently at 1.20, (c/f. 1.17 during the PR), ie. it makes little sense to continue working on that PR. 

Thanks.
